codecov.yml

```diff
@@ -0,0 +1,19 @@
+coverage:
+  round: down
+  precision: 2
+  range: '0...100'
+  status:
+    project:
+      default:
+        # basic
+        target: 0%
```

Comment:
In the How We Work, I recommended 90% for this. All of our FHIR Works repo have more than 90% coverage.

Reply:
That's super helpful, thanks! Was originally just trying to see what coverage we were at but it didn't occur to me to just use yarn.